### PR TITLE
OEUI-233: Combine lab/drug draft orders in the state

### PR DESCRIPTION
**Pull request by IAMOTZ**
### JIRA TICKET NAME
[OEUI-233: Combine the Lab-draft-orders and the Drug-draft-orders](https://issues.openmrs.org/browse/OEUI-233)

### SUMMARY
In the previous implementation of the Draft orders, the lab-draft-orders and the drug-draft-orders are stored in different locations of the state. As part of the implementation to have just one Draft on all the orders page, this Pull Request combines the lab-draft-orders and the drug-draft-orders in the state.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "react-router-dom": "^4.2.2",
     "redux": "^3.6.0",
     "redux-form": "^7.4.2",
-    "redux-promise-middleware": "^4.4.2",
+    "redux-merge-reducers": "0.0.2",
+    "redux-promise-middleware": "^4.2.0",
```
Comment (rhenshaw56) [inline package.json]: @IAMOTZ why the version downgrade of the `redux-promise-middleware` ?

Comment (IAMOTZ) [reply package.json]: That was not intentional, it probably happened when I was doing rebase to fix merge conflicts. I would update it back.
Thanks, @rhenshaw56.

File: .gitignore [hunk 2]
```diff
@@ -5,7 +5,6 @@ config.json
 *.zip
 coverage
 .nyc_output
-package-lock.json
 *.snap
 .DS_Store
 .idea
```
Comment (dondrzzy) [inline .gitignore]: Add a new line please

File: app/js/reducers/draftReducer/draftLabOrderReducer.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  ADD_PANEL_TO_DRAFT_LAB_ORDER,
+  ADD_TEST_TO_DRAFT_LAB_ORDER,
+  DELETE_TEST_FROM_DRAFT_LAB_ORDER,
+  DELETE_PANEL_FROM_DRAFT_LAB_ORDER,
+  DELETE_ALL_ITEMS_IN_DRAFT_LAB_ORDER,
+} from '../../actions/actionTypes';
+import initialState from '../initialState';
+
+
+/* *Utility -> TODO: Move to utility folder */
```
[the remark this answers is not in the document]
Comment (IAMOTZ) [reply app/js/reducers/draftReducer/draftLabOrderReducer.js]: That is to be worked on by @rhenshaw56. 
He made the changes, I just changed the file location with other changes.

**Pull request merged**